lib/rspec/mocks/error_generator.rb

```diff
@@ -50,36 +50,35 @@ def raise_unexpected_message_error(message, *args)
       end
 
       # @private
-      def raise_unexpected_message_args_error(expectation, *args)
+      def message_error(expectation, *args)
```

Comment:
Yup, good catch. the `# @private` must have visually threw me off on that. Making private would be best.

Since this might have arguments from a single call as well, do you think `args_for_calls` would fit better?  That would then cover single and multiple calls and still represent a collection of arguments.  I'm okay with either name, but if a single call was issued and it was called `args_for_multiple_calls`, it might not technically be correct.